ext/openssl/tests/openssl_cms_sign_der.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+openssl_cms_sign() tests
+--SKIPIF--
+<?php if (!extension_loaded("openssl")) print "skip"; ?>
+--FILE--
+<?php
+$infile = __DIR__ . "/cert.crt";
+$outfile = tempnam(sys_get_temp_dir(), "ssl");
+if ($outfile === false) {
+    die("failed to get a temporary filename!");
+}
+
+$privkey = "file://" . __DIR__ . "/private_rsa_1024.key";
+$single_cert = "file://" . __DIR__ . "/cert.crt";
+$assoc_headers = array("To" => "test@test", "Subject" => "testing openssl_cms_sign()");
+$headers = array("test@test", "testing openssl_cms_sign()");
+$empty_headers = array();
+$wrong = "wrong";
+$empty = "";
+
+var_dump(openssl_cms_sign($infile, $outfile, openssl_x509_read($single_cert), $privkey, $empty_headers, OPENSSL_CMS_DETACHED|OPENSSL_CMS_BINARY, OPENSSL_ENCODING_DER));
```

Comment:
Again it would be good to verify that something has been written to the `$outfile`

Reply:
Yes.

Reply:
Ok, I've added an additional test for the next commit.